example/04_gemm_add_add_fastgelu/gemm_add_add_fastgelu_xdl_lds_direct_load_fp32.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
I guess you forgot this line: 
```suggestion
// SPDX-License-Identifier: MIT
// Copyright (c) 2023, Advanced Micro Devices, Inc. All rights reserved.
```

Reply:
yep, thanks for catching that!